ci/slab.toml

```diff
@@ -71,6 +71,12 @@ profile = "m6i-cpu-bench"
 matrix = [0,1,2,3,4,5,6,7,8,9,10]
 max_parallel_jobs = 2
 
+# Trigger ML tests with latest CP
```

Comment:
awesome

by the way, @zama-ai/machine-learning , it looks pretty simple to use Slab, maybe at some point, we'll want to do the same for some specific launched-by-hand tests